contributing.md

```diff
@@ -1,14 +1,51 @@
-#### How to reach us
-The mailing list for developers : systemroles@lists.fedorahosted.org
 
-[Subscribe to the mailing list here](https://lists.fedorahosted.org/admin/lists/systemroles.lists.fedorahosted.org/)
+# Contributing to Network @ Linux System Roles
 
-[Archive of the mailing list](https://lists.fedorahosted.org/archives/list/systemroles@lists.fedorahosted.org/)
+## Where to start
+- **Bugs and needed implementations** are listed on [Github
+  Issues](https://github.com/linux-system-roles/network/issus). Issues labeled
+with **help wanted** are likely to be suitable for new contributors!
+- **Code** is managed on
+  [Github](https://github.com/linux-system-roles/network), using [Pull
+Requests](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests).
 
-If you are using IRC, join the `#systemroles` IRC channel on freenode
+## How to contribute
+
+1. Make a
+   [fork](https://help.github.com/en/github/getting-started-with-github/fork-a-repo)
+of this repository.
+
+2. Create a new git branch on your local fork (the name is not relevant) and
+   make the changes you need to complete an issue.
+
+3. Do not forget to run unit and integration tests before pushing any changes!
+  
+  - This project uses [tox](https://tox.readthedocs.io/en/latest/) to run unit
```

Comment:
Please also mention that `tox` without any arguments will run all tests (except for the molecule tests, as they need docker).

Reply:
Ok, fixed!